make-release.sh

```diff
@@ -37,17 +42,17 @@ check() {
   command -v operator-sdk >/dev/null 2>&1 || { echo -e $RED"operator-sdk is not installed. Aborting."$NC; exit 1; }
 
   local operatorVersion=$(operator-sdk version)
-  [[ ! $operatorVersion =~ .*v0.10.0.* ]] || { echo -e $RED"operator-sdk v0.10.0 is required"$NC; exit 1; }
+  [[ $operatorVersion =~ .*v0.10.0.* ]] || { echo -e $RED"operator-sdk v0.10.0 is required"$NC; exit 1; }
```

Comment:
why this change ?

Reply:
'||' uses to execute the code when the previous command failed so
`[[ $operatorVersion =~ .*v0.10.0.* ]] || {...` will be run when version isn't `v0.10.0`